e2e/parallel/shortcuts-home.test.ts

```diff
@@ -144,26 +144,35 @@ describe.runIf(browser !== 'firefox')(
 
     it('should be able to use arrows to tab switch', async () => {
       await findElementByText(driver, 'Tokens');
-      await executePerformShortcut({ driver, key: 'ARROW_LEFT' });
-      await findElementByText(driver, 'Activity');
       await executePerformShortcut({ driver, key: 'ARROW_RIGHT' });
+      await findElementByText(driver, 'Activity');
+      await executePerformShortcut({ driver, key: 'ARROW_LEFT' });
       await findElementByText(driver, 'Tokens');
       const activity = await isElementFoundByText({
         text: 'Activity',
         driver,
       });
       await executePerformShortcut({ driver, key: 'ARROW_RIGHT' });
+      await executePerformShortcut({ driver, key: 'ARROW_RIGHT' });
       await findElementByText(driver, 'NFTs');
       const tokens = await isElementFoundByText({
         text: 'Tokens',
         driver,
       });
-      await executePerformShortcut({ driver, key: 'ARROW_LEFT' });
+      await executePerformShortcut({ driver, key: 'ARROW_RIGHT' });
+      await findElementByText(driver, 'Points');
       const nfts = await isElementFoundByText({
         text: 'NFTs',
         driver,
       });
-      expect(activity && tokens && nfts).toBe(false);
+      await executePerformShortcut({ driver, key: 'ARROW_LEFT' });
```

Comment:
for this sort of thing you can do this:
```suggestion
      await executePerformShortcut({ driver, key: 'ARROW_LEFT', timesToPress: 3 });
```